.github/workflows/ci-rust.yml

```diff
@@ -89,6 +90,22 @@ jobs:
         with:
           tool: nextest@0.9.54, wasm-pack@0.11.0, cargo-deny@0.14.3
 
+      - uses: ./.github/actions/setup-python-poetry
+        id: setup-python
+        with:
+          poetry-version: ${{ env.poetry-version }}
+          project-path: ./server
+        timeout-minutes: 10
+
+      - name: Test cli
+        shell: bash
+        run: |
+          set -ex
+          python make.py python-dev-install
```

Comment:
Install the python dependencies in a dedicated step with a dedicated timeout.

Reply:
Done